vuepress/docs/next/docs/compose/preinstalled-widgets.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 2
+---
+
+# Preinstalled Widgets
+
+
+The Entando Platform includes a number of useful components to accelerate application development. These consist of widgets, page templates, content templates and content types. This page introduces the CMS, navigation, page, SEO and system widgets that are available out of the box. 
+
+## Widget Attributes
+
+All widgets are required to have the following attributes. These are mandatory regardless of whether a widget is preinstalled, user-created, or inside a bundle.
+
+- `Name`: Specifies the name of the widget
+- `Code`: Satisfies the REGEX `/^[0-9a-zA-Z_.]+$/`
+- `Group`: A group for which the user has "create" permissions
+- `Icon`: Visually represents the widget via either an uploaded icon (SVG file) or one chosen from the icon library
+- `Custom UI`: HTML code that constructs the visual layout of the widget
+- `Config UI`: A JSON structure that informs the widget's configuration. It requires two properties:
```

Comment:
the descriptions for the list seems to change format along the way, where the first few start with verbs but gradually changes sentence format. Might make it more consistent

Reply:
agreed; these are direct from notion and initially i believed that consistency would be at the expense of direct and simplified descriptions, but i'll work with it